bin/spice/cmd/add.go

```diff
@@ -116,15 +144,15 @@ spice add spiceai/quickstart
 			}
 		}
 
-		slog.Info(fmt.Sprintf("added %s\n", relativePath))
+		if connect {
+			slog.Info(fmt.Sprintf("connected to %s\n", relativePath))
```

Comment:
I would leave this as is, because it's not actually connecting, just getting the connection info.